### PR TITLE
Chore upgrade send to 0.19.0

### PR DESCRIPTION
**Pull request by MWein**
# Why

Resolve a medium dependabot alert entitled `send vulnerable to template injection that can lead to XSS`

# How

Upgraded to `send` v0.19.0 in the cli package.
Upgraded to the latest minor version of `express` in the yarnlock.

# Test Plan

All unit tests in @expo/cli pass

image

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: packages/@expo/cli/CHANGELOG.md [hunk 1]
```diff
@@ -100,6 +100,7 @@
 - Throwing an error for DOM components if no webview installed. ([#31974](https://github.com/expo/expo/pull/31974) by [@kudo](https://github.com/kudo))
 - Update Metro type definitions to `metro@0.80.12`. ([#32007](https://github.com/expo/expo/pull/32007) by [@byCedric](https://github.com/byCedric))
 - Remove `json-schema-deref-sync` dependency. ([#32048](https://github.com/expo/expo/pull/32048) by [@kitten](https://github.com/kitten))
+- Chore upgrade send to 0.19.0 ([#32131](https://github.com/expo/expo/pull/32131) by [@MWein](https://github.com/MWein))
```
Comment (byCedric) [inline packages/@expo/cli/CHANGELOG.md:103]:
```suggestion
- Upgrade send to `0.19.0`. ([#32131](https://github.com/expo/expo/pull/32131) by [@MWein](https://github.com/MWein))
```

**Comment by MWein** [comment]
@byCedric can you have another look at this? It's holding up our process because of the medium security alert

**Comment by MWein** [comment]
@byCedric

**Comment by MWein** [comment]
@byCedric @EvanBacon

Can I get traction on this?

**Review by byCedric — APPROVED** [review]

**Pull request closed without merging**